mobile/src/main/java/rs/readahead/washington/mobile/views/base_ui/MetadataBaseLockFragment.kt

```diff
@@ -40,7 +40,7 @@ open class MetadataBaseLockFragment  : Fragment(){
         super.onViewCreated(view, savedInstanceState)
     }
     override fun onResume() {
-        activity.restrictActivity()
+        //activity.restrictActivity()
```

Comment:
This should not be removed, it make sure that the user is not using the app without an unlocked key store.

Reply:
This is called already from MainActivity on which MicFragment is a part of. That's why I said it launches a double lock screen.
Except if we want a double lock screen.